scripts/start.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+if [[ ${ARAGON_CLIENT_PORT} ]]; 
```

Comment:
Wondering if we should keep the same `REACT_APP` prefix, given that all the other env vars use it.

Alternatively, we could change all the other env vars to not use that prefix since we don't need it anymore after moving to parcel.

Reply:
Actually, on a second thought, I have to say I probably should've stayed consistent with how env variables have been named in the past. I didn't know `REACT_APP_*` vars were related to the wrapper client.